src/program_automation.py

```diff
@@ -53,13 +54,21 @@
 def main(dry_run: bool, project_id: str, token: str, url: str) -> None:
     jira_client = jira.client.JIRA(server=url, token_auth=token)
 
-    checks = [
+    epic_checks = [
```

Comment:
Consider using an OrderedDict and looping on the values, as is done in the team_hygiene.py module.

In the future this will allow to put and load the configuration one wishes to use in a config file instead of hardcoding it in the module itself.

Reply:
One reason I didn't do that is that in one case I need to call get_issues and in another I call get_child_issues.

Knowing that, does it change your mind?

Reply:
Revised in 34dd37e.